### PR TITLE
Allow next-page-no-ad in stories

### PR DESCRIPTION
**Pull request by calebcordry**
Allows pubs to put a `next-page-no-ad` attr on an amp-story-page that will prevent ads from showing in between pages. This feature has existed for a while, but it appears I never wrote the proper validation :(

cc/ @newmuis

### CODE REVIEW

**Comment by calebcordry** [comment]
@newmuis moved to reviewer for owners check

File: extensions/amp-story/1.0/test/validator-amp-story-next-page-no-ad.out [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+|          <h3 align-self="start"></h3>
+|        </amp-story-grid-layer>
+|        <amp-story-grid-layer template="horizontal">
+|          <h1></h1>
+|        </amp-story-grid-layer>
+|      </amp-story-page>
+|      <amp-story-bookend src="http://me.com/bookend-config-src.json" layout="nodisplay"></amp-story-bookend>
+|    </amp-story>
+|  </body>
+|  </html>
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/test/validator-amp-story-next-page-no-ad.out]: Nit: eof newline

Comment (calebcordry) [reply extensions/amp-story/1.0/test/validator-amp-story-next-page-no-ad.out]: this file was generated... @Gregable does it matter from the validator perspective?

Comment (Gregable) [reply extensions/amp-story/1.0/test/validator-amp-story-next-page-no-ad.out]: validator should generate the newline, but won't care.

Comment (calebcordry) [reply extensions/amp-story/1.0/test/validator-amp-story-next-page-no-ad.out]: newline added.

**Pull request merged**